src/edu/wright/cs/raiderplanner/controller/TaskController.java

```diff
@@ -241,6 +258,7 @@ public void toggleComplete() {
 			this.completed.setVisible(true);
 			this.canComplete.setVisible(false);
 		}
+		this.unlockSubmit();
```

Comment:
This method returns a boolean.  Why is it being ignored here?

Reply:
Whether it passes or fails is irrelevant. This is used to fix an issue where if a task is being edited, toggling the complete would not unlock the ok button. This is confusing for the user.